Gateway/Request/CustomerDataBuilder.php

```diff
@@ -62,7 +59,14 @@ public function build(array $buildSubject)
         $billingAddress = $order->getBillingAddress();
         $storeId = $order->getStoreId();
         $additionalInformation = $payment->getAdditionalInformation();
-        $request['body'] = $this->adyenRequestsHelper->buildCustomerData([], $customerId, $billingAddress, $storeId, $payment, $additionalInformation);
+        $request['body'] = $this->adyenRequestsHelper->buildCustomerData(
+            $billingAddress,
+            $storeId,
+            $customerId,
+            $payment,
+            $additionalInformation,
+            []
+        );
```

Comment:
Seems like the code fixer misplaced these arguments?
```suggestion
        $request['body'] = $this->adyenRequestsHelper->buildCustomerData(
            [],
            $customerId,
            $billingAddress,
            $storeId,
            $payment,
            $additionalInformation
        );
```

Reply:
Ah nevermind, I see the function signature also changed